config/clusters/2i2c-aws-us/support.values.yaml

```diff
@@ -27,9 +27,7 @@ prometheus:
           hosts:
             - prometheus.aws.2i2c.cloud
     persistentVolume:
-      # Found to be out of space with at 100Gi disk at 2023-07-31, when it was
-      # increased to 250Gi.
-      size: 250Gi
```

Comment:
The growth rate seems like ~100GB/month, so the increase of 250GB gives us ~2.5 month of time then I figure.

Reply:
@consideRatio I think that was mostly due to the small blocksize we had running, and I think this cluster never actually came back from that. If you look at the disk free space graph now:

<img width="957" alt="image" src="https://github.com/2i2c-org/infrastructure/assets/30430/3fd0aeae-255e-4884-822b-3627730c7dbf">

Usage actually *shrunk* by about 100GB after prometheus came back and compacted all those blocks!